posts/solving-parallel-letter-frequency-elixir-part-2-adding-concurrency.md

```diff
@@ -0,0 +1,127 @@
+Exercises on Exercism are small, synthetic, and often seemingly trivial. It’s easy to imagine that experienced practitioners would have nothing to learn from them. However, solving these synthetic problems can push you to learn and apply parts of your language that you may not have explored. This new learning can lead you to solve real world problems more efficiently or in a more expressive way.
+
+[Parallel Letter Frequency](https://exercism.io/tracks/elixir/exercises/parallel-letter-frequency) is a medium difficulty exercise on [Exercism's Elixir Track](https://exercism.io/tracks/elixir) that unpacks a surprising number of interesting lessons. One of those lessons is how easy it is to add concurrency to Elixir code with core features alone, which has clear benefits for anyone writing applications whose performance could be improved by parallelizing tasks. 
+
+This exercise requires you to implement a `Frequency.frequency/2` function that determines letter frequency in a list of strings, with the added challenge of distributing the calculation to a number of worker processes, set with the `workers` argument:
```

Comment:
I think we need something more in the intro about why concurrency and parallelism can be hard, what the pain points of it often are, or why people dread it. This will set the reader up to appreciate the rest of the article more, especially the bit at the end where you point out that the parallelized solution is as easy to read as the sequential one.

Reply:
Good call, I'll try to add something about this.